### DISCUSSION TITLE
DNS zone(s) are not found error

### DISCUSSION THREAD

**Discussion by TheDago** [shibayan/appservice-acmebot#280] [Q&A] [answered]
I've just installed the app, but when I try to add a new certificate I get the following error: 
`Orchestrator function 'AddCertificate_Orchestrator' failed: The orchestrator function 'IssueCertificate' failed: "The activity function 'Dns01Precondition' failed: "DNS zone(s) are not found. DnsNames = <mydnsname>.com". See the function execution logs for additional details.". See the function execution logs for additional details.`
I didn't see any more info in the `AddCertificate_Orchestrator` logs either, just the one above. The app service is Linux based, running a Java app.
Maybe I'm missing something... can you please help me with this?

**Comment by shibayan** [comment, marked as answer]
For App Service on Linux, Azure DNS is required.
https://github.com/shibayan/appservice-acmebot#issuing-a-wildcard-certificate-or-a-certificate-for-linux

**Reply by TheDago** [reply]
I missed that, thank you
